### PR TITLE
[185172789] Use the support email address to send emails from the Helpdesk

### PR DESCRIPTION
**Pull request by Talleyrand333**
## Is this a Feature, Chore or Bug?
- [*] Feature
- [] Chore
- [] Bug

## Clearly and concisely describe the feature, chore or bug.
Use the support email address to send emails from the Helpdesk

## Analysis and design (optional)
Analyse and attach the design documentation

## Solution description
Describe your code changes in detail for reviewers.
Use the support email address to send emails from the Helpdesk
## Is there a business logic within a doctype?
    - [] Yes
    - [*] No

## Output screenshots (optional)
Post the output screenshots, if a UI is affected or added due to this feature.

## Areas affected and ensured
List out the areas affected by your code changes.
HD Ticket 
Email Account

## Is there any existing behavior change of other features due to this code change?
Mention Yes or No. If Yes, provide the appropriate explanation.

## Did you test with the following dataset?
- [*] Existing Data
- [*] New Data

## Was child table created? No
    - [] is attachment required?
        did you test attachment
## Did you delete custom field?
    - [] Yes
    - [*] No
        If yes, did you write a delete patch?

## Is patch required?
- [] Yes
- [*] No
    ## Was the patch test?

## Which browser(s) did you use for testing?
  - [*] Chrome
  - [] Safari
  - [] Firefox

### CODE REVIEW

File: one_fm/one_fm/custom/email_account.json
Comment (mymi14s) [inline one_fm/one_fm/custom/email_account.json:1]: All doctype customization(json) must be done with administrator account.

**Comment by mymi14s** [comment]
@Talleyrand333 use administrator account to export the customization

**Review by mymi14s — CHANGES_REQUESTED** [review]
Fix and notify

**Comment by Talleyrand333** [comment]
@mymi14s , Thank you, this has been resolved.

**Review by mymi14s — APPROVED** [review]

**Pull request merged**